index.html

```diff
@@ -2453,6 +2453,15 @@ <h4>API Specification (Thing Model)</h1>
                     given in this Thing Model
                     are not normative and are provided for information only.
                     </p>
+                    <p>
+                        Note that a <code>contentType</code> is required by the WoT TD 1.1
+                        specification [[wot-thing-description11]] for instances of the
+                        <code>ExpectedResponse</code> class (i.e., the JSON object
+                        corresponding to the key <code>response</code> in a form),
+                        even when the body of the HTTP response is supposed to be empty.
+                        In this case, <code>application/octet-stream</code> can be used to
```

Comment:
I don't think `application/octet-stream` is an appropriate content type to use here.  That is used for binary data to be placed in a file. See:

> The "octet-stream" subtype is used to indicate that a body contains
arbitrary binary data.
> ...
> The recommended action for an implementation that receives an
"application/octet-stream" entity is to simply offer to put the data
in a file, with any Content-Transfer-Encoding undone, or perhaps to
use it as input to a user-specified process.
https://www.iana.org/assignments/media-types/application/octet-stream

The following is not ideal, but at least it isn't indicating something false:
```suggestion
                        In this case, <code>application/x-empty</code> can be used to
```

Reply:
I'd be okay with this change – I suppose there are no optional solutions for this problem :/ What do you think, @mmccool?